Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -747,11 +747,11 @@ Right: the mesh generated after the initialization of all connected components
 
 Note that the example \ref
 Mesh_3/mesh_3D_image_with_custom_initialization.cpp also shows how to
-create an 3D image using the undocumente API of CGAL_ImageIO.
+create a 3D image using the undocumented API of CGAL_ImageIO.
 
 \snippet Mesh_3/mesh_3D_image_with_custom_initialization.cpp Create the image
 
-The code of the function `%random_labeled_image()` is in the example file \ref
+The code of the function `%random_labeled_image()` is in the header file \ref
```

Comment:
Right, that is a header file, but it is in the `examples/Mesh_3` directory, and not in `include/CGAL`. That is why I named it an example file.